src/form/ColorMapInput.tsx

```diff
@@ -115,6 +117,11 @@ const ColorMapInput: React.FC<ColorMapProps & InjectedIntlProps> = (props) => {
   });
   const [showCustomColormapModal, setShowCustomColormapModal] = useState(false);
 
+  const setShowCustomColormapModalWithCallback = (bool:boolean) => {
+    setShowCustomColormapModalProp && setShowCustomColormapModalProp(bool);
+    setShowCustomColormapModal(bool);
+  };
+
```

Comment:
This function is unnecessary if you keep the boolean in the parent component only.